libs/langchain/pyproject.toml

```diff
@@ -138,6 +138,7 @@ anthropic = {version = "^0.3.11", optional = true}
 aiosqlite = {version = "^0.19.0", optional = true}
 rspace_client = {version = "^2.5.0", optional = true}
 upstash-redis = {version = "^0.15.0", optional = true}
+kdbai_client = {version = "^0.1.2", optional = true}
```

Comment:
prefer to only include optional dependencies in pyproject if they're needed for unit testing (the all extras group is deprecated), so no need to add here